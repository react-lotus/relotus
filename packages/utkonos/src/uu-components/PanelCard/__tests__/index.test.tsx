import React from 'react';
import renderer from 'react-test-renderer';
import { PanelCard } from '..';

describe('PanelCard', () => {
  describe('snapshot', () => {
    it('should match PanelCard with Header, Body and Footer', () => {
      const tree = renderer
        .create(
          <PanelCard>
            <PanelCard.Header>
              <PanelCard.Title>Header</PanelCard.Title>
            </PanelCard.Header>
            <PanelCard.Body>Body</PanelCard.Body>
            <PanelCard.Footer>Footer</PanelCard.Footer>
          </PanelCard>,
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });

    it('should match PanelCard with custom header', () => {
      const tree = renderer
        .create(
          <PanelCard>
            <PanelCard.Header>
              <span>Header</span>
            </PanelCard.Header>
          </PanelCard>,
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
