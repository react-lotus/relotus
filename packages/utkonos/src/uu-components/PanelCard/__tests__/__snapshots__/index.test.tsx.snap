// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelCard snapshot should match PanelCard with Header, Body and Footer 1`] = `
<div
  className="uu-panelCard uu-scrollbar"
>
  <div
    className="uu-panelCardHeader"
  >
    <h2
      className="uu-panelCardTitle"
    >
      Header
    </h2>
  </div>
  <div
    className="uu-panelCardBody"
  >
    Body
  </div>
  <div
    className="uu-panelCardFooter"
  >
    Footer
  </div>
</div>
`;

exports[`PanelCard snapshot should match PanelCard with custom header 1`] = `
<div
  className="uu-panelCard uu-scrollbar"
>
  <div
    className="uu-panelCardHeader"
  >
    <span>
      Header
    </span>
  </div>
</div>
`;
